python/py-cli-helpers/Portfile

```diff
@@ -14,25 +16,31 @@ long_description    CLI Helpers is a Python package that makes it easy \
                     to perform common tasks when building command-line apps. \
                     It’s a helper library for command-line interfaces.
 
-python.versions     27 36 37 38
+python.versions     27 36 37 38 39
 
 homepage            https://pypi.python.org/pypi/${python.rootname}/
-master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
 distname            cli_helpers-${version}
```

Comment:
ah, it's failing now with checksum issues for the py27 subport. Since this is a non-default `distname` you'll probably have to copy this in the `if {${python.version} eq 27} {` block

Reply:
It was working for me locally… hmm.

Reply:
nvm you're right

Reply:
well... assumptions are dangerous ;) The CI was quite clear about it; I'd expect that if you would do a `sudo port clean py\*-cli-helpers` locally and then try to install the py27 subport it would fail as well. You probably still had a `distfile` around for one of the py3x subports and that was the one that was being used.